README.md

```diff
@@ -427,6 +446,22 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 **[⬆ Back to Index](#content)**
 
+## Frontend Interview Series
+
+- :video_camera: [Frontend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFXzRwSTSwqL7U2hoMk6K6F)
+
+---
+
+**[⬆ Back to Index](#content)**
+
+## Frontend with HTML CSS JS & Browser APIs
+
+- :video_camera: [Frontend with HTML CSS JS & Browser APIs](https://youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC)
```

Comment:
Move this to the Web Development category that you created below

Reply:
Done!